osu.Game/Screens/Menu/IntroWelcome.cs

```diff
@@ -0,0 +1,149 @@
+                }, delay_step_two);
+
+                LoadComponentAsync(new WelcomeIntroSequence
+                {
+                    RelativeSizeAxes = Axes.Both
+                }, AddInternal);
+            }
+        }
+
+        public override void OnSuspending(IScreen next)
+        {
+            this.FadeOut(300);
+            base.OnSuspending(next);
+        }
+
+        private class WelcomeIntroSequence : Container
+        {
+            private Sprite welcomeText;
+
+            [BackgroundDependencyLoader]
+            private void load(TextureStore textures)
+            {
+                Origin = Anchor.Centre;
+                Anchor = Anchor.Centre;
+                Children = new Drawable[]
+                {
+                    new Container
+                    {
+                        Anchor = Anchor.Centre,
+                        Origin = Anchor.Centre,
+                        AutoSizeAxes = Axes.Both,
+                        Children = new Drawable[]
+                        {
+                            new Container
+                            {
+                                AutoSizeAxes = Axes.Both,
+                                Children = new Drawable[]
+                                {
+                                    new LogoVisualisation
+                                    {
+                                        RelativeSizeAxes = Axes.Both,
+                                        Anchor = Anchor.Centre,
+                                        Origin = Anchor.Centre,
+                                        Alpha = 0.5f,
+                                        AccentColour = Color4.DarkBlue,
+                                        Size = new Vector2(0.96f)
+                                    },
+                                    new Container
+                                    {
+                                        AutoSizeAxes = Axes.Both,
+                                        Children = new Drawable[]
+                                        {
+                                            new Circle
+                                            {
+                                                Anchor = Anchor.Centre,
+                                                Origin = Anchor.Centre,
+                                                Size = new Vector2(480),
+                                                Colour = Color4.Black
+                                            }
+                                        }
+                                    }
+                                }
+                            }
+                        }
+                    },
+                    welcomeText = new Sprite
+                    {
+                        Anchor = Anchor.Centre,
+                        Origin = Anchor.Centre,
+                        Scale = new Vector2(0.3f),
+                        Width = 750,
```

Comment:
width and height specification should not be required. it's automatic when you set a texture.

Reply:
```csharp
Width = 750,
Height = 78,
```
This is hardcoded like this to prevent the game from proportionally scaling it. The original dimensions for the file are `750*156`

`welcomeText.ResizeHeightTo(welcomeText.Height * 2, 500, Easing.In);` resizes the height from `750*78` to `750*156`